opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -102,6 +103,34 @@ def get_context(self) -> 'SpanContext':
             A :class:`.SpanContext` with a copy of this span's immutable state.
         """
 
+    def set_attribute(self: 'Span',
+                      key: str,
+                      value: 'types.AttributeValue'
+                      ) -> None:
+        """Sets an Attribute.
+
+        Sets a single Attribute with the key and value passed as arguments.
+        """
+
+    def add_event(self: 'Span',
+                  name: str,
+                  attributes: 'types.Attributes',
+                  ) -> None:
+        """Adds an Event.
+
+        Adds a single Event with the name and, optionally, attributes passed
```

Comment:
```suggestion
        Adds a single `Event` with the name and, optionally, attributes passed
```

Reply:
At the moment, we don't. We have the following
```
Event = namedtuple('Event', ('name', 'attributes'))
```
But that's in the SDK, not exposed in the API.